CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ IMPROVEMENTS:
 * secrets/gcp: Add check for 403 during rollback to prevent repeated deletion calls [[GH-97](https://github.com/hashicorp/vault-plugin-secrets-gcp/pull/97)]
 * core: Disable usage metrics collection on performance standby nodes. [[GH-9966](https://github.com/hashicorp/vault/pull/9966)]
 * credential/aws: Added X-Amz-Content-Sha256 as a default STS request header [[GH-10009](https://github.com/hashicorp/vault/pull/10009)]
+* core (enterprise): New metrics covering leader status for nodes
```

Comment:
The categories should be in alphabetical order.

But this is the 1.5.4 section, it should be in "NEXT"?

Reply:
Absolutely, put it as an improvement for NEXT / 1.5.5